src/main/java/edu/cornell/kfs/concur/service/impl/ConcurAccessTokenServiceImpl.java

```diff
@@ -21,20 +31,57 @@
 import com.sun.jersey.api.client.config.DefaultClientConfig;
 
 import edu.cornell.kfs.concur.ConcurConstants;
-import edu.cornell.kfs.concur.ConcurPropertyConstants;
+import edu.cornell.kfs.concur.ConcurUtils;
 import edu.cornell.kfs.concur.rest.xmlObjects.AccessTokenDTO;
 import edu.cornell.kfs.concur.service.ConcurAccessTokenService;
 import edu.cornell.kfs.sys.CUKFSConstants;
-import edu.cornell.kfs.sys.businessobject.WebServiceCredential;
 import edu.cornell.kfs.sys.service.WebServiceCredentialService;
 
 public class ConcurAccessTokenServiceImpl implements ConcurAccessTokenService {
-    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurAccessTokenServiceImpl.class);
+    private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurAccessTokenServiceImpl.class);
 
+    private String concurRequestAccessTokenURL;
     private String concurRefreshAccessTokenURL;
+    private String concurRevokeAccessTokenURL;
     protected WebServiceCredentialService webServiceCredentialService;
 
     @Override
+    public void requestNewAccessToken() {
```

Comment:
I've left this name as-is for now, due to the way that I refactored things in my latest revisions. The revoke-and-replace naming has been moved to a different method as a result. Let me know if you think this still needs renaming.